content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,381 @@ permissions:
-not have to store permanent credentials in a secret.
+cert-manager needs an [AWS access key](https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#access_key),
+to authenticate to the Route53 API.
+An access key is defined by AWS as follows:
+> **access key**:
+> The combination of an access key ID (for example, `AKIAIOSFODNN7EXAMPLE`) and a secret access key (for example, `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`). You use access keys to sign API requests that you make to AWS.
+
+You have two options:
+1. (Legacy) Use an [IAM User and a long-term access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+2. (Best Practice) Use an [IAM Role with temporary security credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#bp-workloads-use-roles).
+
+Using an IAM Role with temporary security credentials is considered best practice because:
+1. You do not have to store the long-term access key (e.g. in a Secret)
+2. You don't have to manage [access key rotation](https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#keyrotate).
+
+cert-manager supports multiple ways to get the access key and these can be categorized as either "ambient" or "non-ambient".
+
+### Ambient Credentials
+
+Ambient credentials are credentials which are made available in the cert-manager controller Pod by one of the following mechanisms:
+- [**EKS Pod Identity**](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html):<br/>
+  where cert-manager gets credentials from an [EKS Auth API which runs on every Kubernetes node](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-how-it-works.html).
+- [**EKS IAM Roles for Service Accounts (IRSA)**](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html):<br/>
+  where cert-manager uses a Kubernetes ServiceAccount token which is [mounted into the cert-manager controller Pod](https://docs.aws.amazon.com/eks/latest/userguide/pod-configuration.html).
+- [**EC2 Instance Metadata Service (IMDS)**](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-metadata-security-credentials.html):<br/>
+  where cert-manager gets credentials from the `iam/security-credentials/<role-name>` endpoint of IMDS.
+- [**Environment variables**](https://docs.aws.amazon.com/sdkref/latest/guide/environment-variables.html):<br/>
+  where cert-manager loads credentials from `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables
+  in the cert-manager controller Pod, if those variables are present.
+- [**Shared config and credentials files**](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html):<br/>
+  where cert-manager loads credentials from files (`~/.aws/config` and `~/.aws/credentials`) which are mounted into the cert-manager controller Pod.
+
+The advantage of ambient credentials is that they are easier to set up, well
+documented, and AWS provides ways to automate the configuration.
```

Comment:
From https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#non-ambient-credentials I don't see any blockers to automate non-ambient credentials as well

Reply:
I removed those words. What I meant was that AWS provides **helpers** which inject the necessary environment variables into the cert-manager Pod, rather than the application team having to configure everything in the issuer.
But that's already covered by "easier to set up".